code-challenges.js

```diff
@@ -28,3 +59,32 @@ var hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
+
+const isFullHouse = (array) => {
+  // need temp booleans if we find 3 or 2 instances
+  let foundThree = false
+  let foundTwo = false
+  let arrayLength = 0
+  //map through the array and look for the number of occurances
+  //check all posible digits 0..9
+  for (let i = 0; i < 10; i++) {
+    //find the number of occurrances by checking the length of the filtered array
+    arrayLength = array.filter(value => value==i).length
```

Comment:
at first I was confused about the filter inside of the loop but it makes sense to me now! there is probably a more "efficient" way to solve this but it works! great job